src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const pictures = document.querySelectorAll('.list-item__link');
+const valide = document.getElementById('largeImg');
+
+for (const picture of pictures) {
+  picture.addEventListener('click', (even) => {
```

Comment:
Change the parameter name 'even' to 'event' for better readability. Or it's usually used as just `e`